src/ContainerRegistry/ContainerRegistry/Commands/ConnectAzureContainerRegistry.cs

```diff
@@ -60,6 +64,16 @@ protected override void InitDebuggingFilter()
 
             string LoginScript = string.Format("'{2}' | docker login {0} -u {1} --password-stdin", this.RegistryDataPlaneClient.GetEndPoint(), this.UserName, this.Password);
             WriteObject(this.ExecuteScript<object>(LoginScript));
+            if (ExposeToken) {
+                WriteWarning("You can perform manual login using the provided access token below, for example: 'docker login <loginServer> -u 00000000-0000-0000-0000-000000000000 -p <accessToken>'");
+                var cred = new
+                {
+                    loginServer = Name + ".azurecr.io",
+                    accessToken = this.RegistryDataPlaneClient.Authenticate()
+
+                };
+                WriteObject(JsonConvert.SerializeObject(cred, Formatting.Indented));
```

Comment:
This command will output 2 objects when ExposeToken is specified. Is that expected?